mitodl/mitxpro
# Added proper login handling of app context

#### Pre-Flight checklist


#### What are the relevant tickets?
Part of #73 
Fixes #74 

#### What's this PR do?
Adds app_context endpoint and updates the app to load this and the current user (who may be anonymous). This should allow us to move away from a static global `SETTINGS` object.

#### How should this be manually tested?
You should be able to login still, and once you login your name and a logout link should appear on the page as logged in. This should still show up if you reload.
